### PR TITLE
CAMEL-18123 Aws2-sqs: Operations Purge and Delete requires header

### PR DESCRIPTION
**Pull request by JiriOndrusek**
fixes https://issues.apache.org/jira/browse/CAMEL-18123

### CODE REVIEW

File: components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/AmazonSQSClientMock.java [hunk 1]
```diff
@@ -67,10 +67,15 @@ public class AmazonSQSClientMock implements SqsClient {
     private Map<String, CreateQueueRequest> queues = new LinkedHashMap<>();
     private Map<String, ScheduledFuture<?>> inFlight = new LinkedHashMap<>();
     private ScheduledExecutorService scheduler;
+    private String queueuName = null;
```
Comment (oscerd) [inline components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/AmazonSQSClientMock.java:70]: a little typo

**Review by oscerd — APPROVED** [review]
Looks good, just a minor finding.

**Comment by oscerd** [comment]
Please also amend the commit message with the correct JIRA Issue.

**Comment by JiriOndrusek** [comment]
yes, I'll fix it, thanks

**Review by davsclaus — APPROVED** [review]

**Pull request merged**